code/game/objects/items/blueprints.dm

```diff
@@ -216,6 +218,7 @@
 		A.setup(str)
 		A.contents += turfs
 		A.SetDynamicLighting()
+	A.has_gravity = old.has_gravity
```

Comment:
rare, but what if you managed to remove every instance of an Area? just check old exists first please.